prickly-pufferfish/arena/battle/views.py

```diff
@@ -0,0 +1,95 @@
+from django.views.generic import DetailView, RedirectView
+from django.views.generic.detail import SingleObjectMixin
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+from .models import Battle, Challenger
+
+import os
+import random
+
+
+class RandomQuestion():
+    def question(self):
+        path = "python_questions"
+        append_string = ''
+        list_of_strings = []
+        names = os.listdir(path)
+        # removes hidden files, just in case.
+        names = [f for f in names if not f.startswith('.')]
+        list_of_paths = []
+        for root, dirs, files in os.walk(path):
+            for f in files:
+                x = os.path.join(root, f)
+                list_of_paths.append(x)
+
+        question = random.choice(list_of_paths)
+        open_question = open(question, 'r')
+        display_question = open_question.read()
+        display_question = display_question[3:-4]
+
+        for i in display_question:
+            if i != '\n':
+                append_string += i
+            if i == '/':
+                list_of_strings.append(append_string[0:-1])
+                append_string = ""
+        return list_of_strings
+
+    def name(self):
+        path = "python_questions"
+        names = os.listdir(path)
+        # removes hidden files, just in case.
+        names = [f for f in names if not f.startswith('.')]
+        name = random.choice(names)
+        return name
+
+
+my_class = RandomQuestion()
```

Comment:
Is this specifically used anywhere?  Could also use a more descriptive variable name.